src/modules/auth/resources/ResourcesDeletedDetailContainer.js

```diff
@@ -0,0 +1,167 @@
+			setResource(response.data);
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Can't fetch resource details")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+	// Set delete resource dialog
+	const confirmForm = (type) => {
+		var r = confirm(t(`ResourcesDeletedDetailContainer|Do you really want to ${type === "delete" ? 'hard-delete' : 'retrieve'} this resource`));
+		if (r == true) {
+			if (type === "delete") {
+				hardDelete();
+			} else {
+				retrieveResource();
+			}
+		}
+	}
+
+	// Retrieve the resource
+	const retrieveResource = async () => {
+		try {
+			let response = await SeaCatAuthAPI.post(`/resource/${resource_id}`, {});
+			if (response.data.result !== "OK") {
+				throw new Error(t("ResourcesDeletedDetailContainer|Failed to retrieve resource"));
+			}
+			props.app.addAlert("success", t("ResourcesDeletedDetailContainer|Resource retrieved successfuly"));
+			props.history.push("/auth/resources");
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Failed to retrieve resource")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+	// Hard-deletes selected resource. After this action, the resource cannot be retrieved anymore.
+	const hardDelete = async () => {
+		try {
+			let response = await SeaCatAuthAPI.delete(`/resource/${resource_id}`, { params: { hard_delete: true } });
+			if (response.data.result !== "OK") {
+				throw new Error(t("ResourcesDeletedDetailContainer|Failed to hard-delete this resource"));
+			}
+			props.app.addAlert("success", t("ResourcesDeletedDetailContainer|Resource was successfully hard-deleted"));
+			props.history.push("/auth/resources");
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Failed to hard-delete this resource")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+
+	return (
+
+		<Container>
+			<Row className="mb-4 justify-content-md-center">
+				<Col md={8}>
+					<Card>
+						<CardHeader className="border-bottom">
+							<div className="card-header-title">
+								<i className="cil-lock-locked pr-2"></i>
+								{t("ResourcesDeletedDetailContainer|Resource")}
+							</div>
+						</CardHeader>
+
+						<CardBody>
+							<Row className="card-body-row">
+								<Col md={3}>{t("Name")}</Col>
+								<Col>{resource && resource._id}</Col>
+							</Row>
+							<Row className="mt-3 card-body-row">
+								<Col md={3}>{t("Created at")}</Col>
+								<Col><DateTime value={resource && resource._c} /></Col>
+							</Row>
+							<Row className="card-body-row">
+								<Col md={3}>{t("Modified at")}</Col>
+								<Col><DateTime value={resource && resource._m} /></Col>
+							</Row>
+							<Row className="card-body-row">
+								<Col sm={3}>{t("Description")}</Col>
+								<Col sm={6} style={{minHeight: "90px"}}>{resource &&  resource.description }</Col>
```

Comment:
@petrKavulok Styling please to the scss file. Try to avoid inline styling in overall

Reply:
i tried to go the 'lazy way' but you're right, thank you